src/indexer/index.ts

```diff
@@ -39,6 +40,11 @@ export const createIndexer = (legacyTemplate: boolean): Indexer => ({
         throw new LegacyTemplateNotEnabledError(filename);
       }
 
+      // WARN: We can't use instance of `StorybookSvelteCSFError`, because is an _abstract_ class :sob:
+      if (isStorybookSvelteCSFError(error)) {
+        throw error;
+      }
+
       throw new IndexerParseError();
```

Comment:
you might want to pass in cause here to help understand where it originated. completely swallowing the og error and throwing a non-descriptive one requires bringing a debugger to inspect... https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause